src/registrar/templates/includes/domains_table.html

```diff
@@ -167,6 +178,18 @@ <h2>Status</h2>
     <div class="domains__no-search-results display-none">
       <p>No results found</p>
     </div>
+
+    {% if not portfolio %}
+    <div class="grid-row">
```

Comment:
Put your ifs around the div and close div markups, don't repeat the button markup. Haven't looked at this in a browser so hold off a bit.